doc/gh-pages/notation/builtin_macros.md

```diff
@@ -8,9 +8,26 @@ nav_order: 9
 # Built-In Macros
 
 {: .story }
-> TODO
+>**GM**: You fail to secure yourself on the wall and slip towards the spikes. While falling, you hit your head against the wall. 
+>
+>**Grindon**: Gosh, that hurts.
+>
+>**GM**: You miraculously hit the ground without dying. But your blow to the head makes you delirious.
+>
+>**Grindon**: Oh look, fairies wearing armors painted with Zodiac signs!
+>
+>**GM**: Yes... (sighs) As they fligh around your head, you decide to eat one of them.
+>
+>**Grindon**: Okay that's a #PLANETS die.
+>
+>*GNOLL*: [Mercury through Pluto]
+>
+>**Grindon The Brave**: Well... not my favorite taste.

```

Comment:
```suggestion
>*GNOLL*: [Neptune]
>
>**Grindon The Brave**: Well... not my favorite flavour...

```
This is my mistake for being unclear in the rest of the documentation. When I wrote 'mercury through pluto' i meant 'all the planets from mercury all the way to pluto'. I was being lazy!
So just picking one is the way to go!

Reply:
Ok. I will pick one. Let me secretly select it and check its English name.
I will also update the actual doc with your improved sentence.